tests/other/no-exceptions/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+#===============================================================================
+# Copyright 2016-2018 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#===============================================================================
+
+if (NOT DNNL_BUILD_EXAMPLES)
```

Comment:
Do we really need this check? I see that `tests/CMakeLists.txt` already contains the `DNNL_BUILD_TESTS` check at the top.